kore/src/Kore/Repl.hs

```diff
@@ -147,38 +155,34 @@ runRepl tools simplifier predicateSimplifier axiomToIdSimplifier axioms' claims'
             }
 
     firstClaim :: Claim level
-    firstClaim = maybe (error "No claims found") id . listToMaybe $ claims'
+    firstClaim = maybe (error "No claims found") id $ listToMaybe claims'
 
     firstClaimExecutionGraph :: ExecutionGraph level
     firstClaimExecutionGraph = emptyExecutionGraph firstClaim
 
     stepper0 :: StateT (ReplState level) Simplifier Bool
     stepper0 = do
-        ReplState
-            { claims
-            , axioms
-            , graph
-            , claim
-            , node
-            } <- get
+        ReplState { claims , axioms , graph , claim , node } <- get
         if Graph.outdeg (Strategy.graph graph) node == 0
             then do
-                graph' <- lift . catchInterruptWithDefault graph
-                    $ verifyClaimStep
-                        tools
-                        simplifier
-                        predicateSimplifier
-                        axiomToIdSimplifier
-                        claim
-                        claims
-                        axioms
-                        graph
-                        node
+                graph' <-
+                    lift
+                        . catchInterruptWithDefault graph
+                        $ verifyClaimStep
+                            tools
+                            simplifier
+                            predicateSimplifier
+                            axiomToIdSimplifier
+                            claim
```

Comment:
Could we pass the `ReplState` directly to `verifyClaimStep` instead of passing its fields one by one?

Reply:
I don't like the idea of having any module depend on anything related to the REPL right now, so I would avoid this.

Reply:
I didn't observe that `verifyClaimStep` comes from outside the module. Sounds good to me!